src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -68,6 +69,21 @@ export default App.extend(extend({
       Radio.request('entities', 'fetch:files:collection:byAction', this.action.id),
     ];
   },
+  onWebsocketMessage({ category, author, payload }) {
```

Comment:
I think this needs to be on the action somehow.  we're talking about having a comment count.. possibly having these show up in a notification bar.. this'd require the action sidebar be open to get a new comment.  I think we'll have to trigger some event on the action we can listento here for just the `this.activityCollection.add(model);`

Reply:
Took a crack at implementing this.